demos/projects/ESPRESSIF/aziotkit/components/azure-iot-kit-sensors/src/sensor_manager.c

```diff
@@ -0,0 +1,291 @@
+#include <stdio.h>
+#include <math.h>
+#include "driver/i2c.h"

```

Comment:
>driver [](http://example.com/codeflow?start=10&length=6)

Few points in include files:
1. Implementation should always take the header file for which it is implmenting interface as a first header file.
2. then all standard header files.
3. then project specific .

with each section seperated by space.

One reference : 
In dir/foo.cc or dir/foo_test.cc, whose main purpose is to implement or test the stuff in dir2/foo2.h, order your includes as follows:

dir2/foo2.h.
A blank line
C system headers (more precisely: headers in angle brackets with the .h extension), e.g., <unistd.h>, <stdlib.h>.
A blank line
C++ standard library headers (without file extension), e.g., <algorithm>, <cstddef>.
A blank line
Other libraries' .h files.
Your project's .h files.

[https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)